### ISSUE TITLE
Add list of organisational members to home page

### ISSUE THREAD

**Issue by henare** [AOGPN/opengovernment.org.au#127]
Peter Timmins asked if we could do this when I last saw him. IIRC the idea is to show that the network has the weight of a number of organisations behind it so that other organisations will be more likely to join.

**Comment by kat** [comment]
Excellent, do you see that being like the way we display 'supporting organisations' like this? 
![screen shot 2016-04-18 at 12 29 02 pm](https://cloud.githubusercontent.com/assets/95376/14592381/3d30b0fc-0561-11e6-88c1-2b2e91a02a6e.png)

**Comment by henare** [comment]
@kat yes, that's what I suggested to Peter. He didn't want the logos yet for some reason - hopefully he can chime in here with the reason.

**Comment by kat** [comment]
Curious. Its such a super easy way to recognise things we know (for those of us who are very visual at least, which is the majority of people) Will be interesting to understand this issue better.

**Comment by petert10** [comment]
If its easy OK-just looking for an easy step up. 
The current organisation members are
Accountability Roundtable,Transparency International Australia, Open Knowledge Australia, OpenAustralia Foundation, Electronic Frontiers Australia, Code for Australia, Australian Press Council, Media Entertainment and Arts Alliance, Publish What You Pay, Blueprint for Free Speech, Creative Commons Australia, Australian Privacy Foundation and Internet Australia.

**Comment by kat** [comment]
@henare @equivalentideas I could grab the logos and resize to a standard size this afternoon for this.

**Comment by petert10** [comment]
Thanks. Not sure if the list I gave you included the latest- Australian
Policy Online. P.
http://apo.org.au/

**Comment by kat** [comment]
Hi @petert10 @henare @equivalentideas  I've grabbed all the logos of non-profit Civil Society organisations above. Although you mention APO.org.au above, interestingly they self identified as a business when they registered. On their site 'Policy Online is a not-for-profit, open access digital library for researchers, policy makers and practitioners to discover, share and manage public knowledge resources.' I didn’t find any record of them on the Australian Business Register, not for profit or otherwise - so it looks like they're an unregistered coalition of partner organisations. Their domain was registered by someone from a partnering university.  

Code for Australia as well as others registered via google form, Thomson Reuters Financial & Risk, the Ethicos Group & Engage2 rare all registered for profit businesses according to the Australian Business Register, although couple of the businesses with civil society interests (Ethicos and Engage2) self identified as Civil Society organisations, they're pretty clearly businesses and not Civil Society Organisations.

**Comment by equivalentideas** [comment]
>  I've grabbed all the logos of non-profit Civil Society organisations above.

@kat could you put these in the Google Drive folder, or link to where they are please? :)

**Comment by equivalentideas** [comment]
Here's the logos collected so far: https://drive.google.com/drive/u/0/folders/0B_0JrLy9bmIPLVM3bFRVMkh3dk0

Thanks @kat

**Comment by kat** [comment]
Have added them to a (draft) page called Member Organisations
